### DISCUSSION TITLE
Can xpore be used for analysis Promethion direct rna sequence data

### DISCUSSION THREAD

**Discussion by gaoyubang** [GoekeLab/xpore#132] [Q&A] [answered]
Dear  Ploy N. Pratanwanich:

     I'm processing my Promethion nanopore direct rna sequence data with xpore. Can xpore be used for analysis Promethion direct rna sequence data?
     Looking forward to your reply！

Thanks!

**Comment by jonathangoeke** [comment, marked as answer]
Hi @gaoyubang 
Yes, xPore can analyse Promethion direct RNA-Seq data.
Thanks!
Jonathan
